src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -592,12 +572,13 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
     }
 
     /* All empty lines or lines with only white spaces will be skipped.
-     * The invalidity info returned are in HTML format.
+     * The invalidity info is in HTML format.
      */
-    private List<String> getInvalidityInfoInEnrollLines(String lines, String courseId) throws EnrollException {
+    public List<StudentAttributes> createAndValidateStudents(String lines, String courseId) throws EnrollException {
```

Comment:
Well, this new method combined the part on validation and creating studentList together(done separately in the original code). Also, mentioning Validate might help with understanding the purpose of this function. Meanwhile, I have explained the exception in the header comment to mention about validating. Any suggestion on this ?@wkurniawan07